src/main/java/net/pms/dlna/MapFile.java

```diff
@@ -75,6 +78,18 @@ public MapFile(MapFileConfiguration conf, List<File> list) {
 		forcedName = null;
 	}
 
+	protected boolean isFolderThumbnail(File file) {
+		if (file == null) {
+			return false;
+		}
+		String fileName = file.getName().toLowerCase(Locale.ROOT);
+		String extension = FileUtil.getExtension(fileName);
+		if (!"jpg".equals(extension) && !"png".equals(extension)) {
```

Comment:
We reference "jpg" and "png" here and on line 214 and others, I wonder if we could use a constant like `ALLOWED_FOLDER_IMAGE_EXTENSIONS` to increase cohesiveness